### PR TITLE
Update all docs links in product After JS and Node merge

### PR DESCRIPTION
**Pull request by smeubank**
After this PR all of the links including node as a platform need to be updated

https://github.com/getsentry/sentry-docs/pull/9710

Example:

image

leads to:

image

### CODE REVIEW

**Comment by smeubank** [comment]
maybe some routing rules in docs could cover other places where such links exist?

File: static/app/components/onboarding/gettingStartedDoc/utils/metricsOnboarding.tsx [hunk 1]
```diff
@@ -217,7 +217,7 @@ export const getJSServerMetricsOnboarding = (): OnboardingConfig => ({
             'Learn more about metrics and how to configure them, by reading the [docsLink:docs].',
             {
               docsLink: (
-                <ExternalLink href="https://docs.sentry.io/platforms/node/metrics/" />
+                <ExternalLink href="https://docs.sentry.io/platforms/javascript/metrics/" />
```
Comment (ArthurKnaus) [inline static/app/components/onboarding/gettingStartedDoc/utils/metricsOnboarding.tsx:220]:
```suggestion
                <ExternalLink href="https://docs.sentry.io/platforms/javascript/guides/node/metrics/" />
```

**Comment by ArthurKnaus** [comment]
There are a few links that now link to the wrong pages

**Comment by smeubank** [comment]
> There are a few links that now link to the wrong pages

yep you're right, I was catching that on the static-sites will dluble check here again 🙇

**Review by smeubank — COMMENTED** [review]
@ArthurKnaus should be good now, i fixed a few more and some formatting stuff, i know this is kind of annoying one. Thanks for checking!

**Review by ArthurKnaus — APPROVED** [review]
lgtm 🚀

**Pull request merged**